### ISSUE TITLE
Rendering AR background broken by update to 3.1.0-preview1

### ISSUE THREAD

**Issue by kjyv** [Unity-Technologies/arfoundation-samples#390]
The changelog for ARFoundation 3.1.0-preview1 states:
> Fixed an issue which could cause the background camera texture to stop functioning correctly on certain devices running OpenGLES3

It seems this has however broken the rendering of the background on my normal iOS Metal build, it is always black/previous content (3D objects are visible). Reverting to the 3.0.1 version makes it work again.

**Comment by tdmowrer** [comment]
That seems unlikely to be the culprit as there should be no difference on other platforms; the new code is gated by
```csharp
if (SystemInfo.graphicsDeviceType == GraphicsDeviceType.OpenGLES3)
```

The current samples which depend on 3.1.0-preview.4 work fine for me. Are you sure it isn't a different issue with the upgrade? Are there any errors in the Xcode console?

**Comment by kjyv** [comment]
It appears to work now with preview.4. I saw that also in 3.1.0-preview.3 some mistake about background rendering was fixed, I'm not using LWRP though. Maybe that fixed it. You seem to have bad release versioning btw. Why was the occlusion manager renamed/removed but not included in the non-preview version 3.0.1. Why are such fixes not done in 3.0.2 or similar instead of 3.1.0-previews. There should be a useable distinction between stable (fixes) and preview (new features). Otherwise just tell people that you don't have a stable version and it's always rolling.

**Comment by tdmowrer** [comment]
> Why was the occlusion manager renamed/removed but not included in the non-preview version 3.0.1

Non preview packages are "verified", which must meet a higher quality standard. We didn't feel the occlusion feature was production ready yet, so it is only in the preview packages.

> Why are such fixes not done in 3.0.2 or similar instead of 3.1.0-previews.

The fix for the OpenGLES rendering issue is in the 3.0.1 verified release, and it is mentioned in the [changelog](https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@3.0/changelog/CHANGELOG.html#fixes-1) for the final preview of the 3.0 package. The fact that it also appears in the changelog for a 3.1 preview release seems to just be a mistake in the log.

> I saw that also in 3.1.0-preview.3 some mistake about background rendering was fixed

That was not discovered until after 3.0 was released, and it takes more time to backport it to verified versions. Verified packages are shipped with the Editor and are part of the Unity installer package.

> There should be a useable distinction between stable (fixes) and preview (new features).

This is a good point and there's a lot of discussion on this topic for packages in general, not just ARFoundation. This is valuable feedback; thanks!
